### PR TITLE
fixed issues with the learn-more-button

### PR DESCRIPTION
**Pull request by celinehuang**
Closes https://github.com/plotly/dash-sample-apps/issues/132

### CODE REVIEW

**Comment by shammamah** [comment]
@celinehuang Please replace the entire PR comment with "Closes #...", with '...' replaced with the issue number that this PR addresses!

File: apps/dash-tsne/demo_intro.md [hunk 1]
```diff
@@ -1 +1 @@
-The Scatter plot above is the result of running the t-SNE algorithm on the MNIST digits, resulting in a 3D visualization of the image dataset. For demo purposes, all the data were pre-generated using limited number of input parameters, a subset of 3000 samples, and displayed instantly. To train the algorithm locally, in-browser, and using your own datasets, follow the instructions on the project repo to setup the local version. To learn more about how the t-SNE Explorer works, click on "Learn More" below.
+The Scatter plot below is the result of running the t-SNE algorithm on the MNIST digits, resulting in a 3D visualization of the image dataset. For demo purposes, all the data were pre-generated using limited number of input parameters, a subset of 3000 samples, and displayed instantly. To train the algorithm locally, in-browser, and using your own datasets, follow the instructions on the project repo to setup the local version. To learn more about how the t-SNE Explorer works, click on "Learn More" below.
```
Comment (shammamah) [inline apps/dash-tsne/demo_intro.md]:
```suggestion
The scatter plot below is the result of running the t-SNE algorithm on the MNIST digits, resulting in a 3D visualization of the image dataset. For demo purposes, all the data were pre-generated using limited number of input parameters, a subset of 3000 samples, and displayed instantly. To train the algorithm locally, in-browser, and using your own datasets, follow the instructions on the project repo to setup the local version. To learn more about how the t-SNE Explorer works, click on "Learn More" below.
```

File: apps/dash-tsne/runtime.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-python-3.6.6
\ No newline at end of file
+python-3.6.6
```
Comment (shammamah) [inline apps/dash-tsne/runtime.txt]: Can you remove the newline here? This diff isn't super meaningful, so it shouldn't be in the PR.

**Comment by celinehuang** [comment]
@shammamah do you think I should redo the screenshots with removed the dash debugger on the bottom right corner? Also, should I keep the gif, I thought the quality isn't as good as the screenshots.

**Comment by shammamah** [comment]
>do you think I should redo the screenshots with removed the dash debugger on the bottom right corner? 

Yes, please -- good catch :) 

>Also, should I keep the gif, I thought the quality isn't as good as the screenshots.

Is it possible to have a higher-quality gif?

**Comment by celinehuang** [comment]
> > do you think I should redo the screenshots with removed the dash debugger on the bottom right corner?
> 
> Yes, please -- good catch :)
> 
> > Also, should I keep the gif, I thought the quality isn't as good as the screenshots.
> 
> Is it possible to have a higher-quality gif?

I'm going to push the changed code then try recording from the gallery app, if it doesn't look any better, should I remove the gif?

**Comment by shammamah** [comment]
@celinehuang Yep, please do that!

**Comment by celinehuang** [comment]
> @celinehuang Yep, please do that!

@shammamah Let me know what you think of the gif on the README, otherwise I think everything should be good!

**Comment by shammamah** [comment]
@celinehuang For the gif, is it possible to include the digit that displays to the right of the graph? With Giphy you can take a screen capture of the entire window.

**Comment by ronniebugia** [comment]
💃 Looks good

**Pull request merged**